src/rules/dollar-variable-colon-space-before/index.js

```diff
@@ -0,0 +1,34 @@
+import {
+  namespace,
+  whitespaceChecker,
+} from "../../utils"
+import { utils } from "stylelint"
+import { variableColonSpaceChecker } from "../dollar-variable-colon-space-after"
```

Comment:
hmm.. Should `variableColonSpaceChecker` be refactored out of the rule? This creates tight coupling between the rules.